dao/seeds/source_types.yml

```diff
@@ -66,6 +66,24 @@ amazon:
           pattern: "^arnaws:.*"
         - type: min-length
           threshold: 10
+    - type: provisioning-arn
+      name: Provisioning ARN
+      fields:
+        - component: text-field
+          name: authentication.authtype
+          hideField: true
+          initializeOnMount: true
+          initialValue: provisioning-arn
+        - name: authentication.username
+          component: text-field
+          label: ARN
+          isRequired: true
+          validate:
+            - type: required
+            - type: pattern
+              pattern: "^arnaws:.*"
+            - type: min-length
+              threshold: 10
```

Comment:
These just need moved 2 spaces to the left - fields is an array of objects.

Reply:
I've reduced the indentation in two spaces. I was confused by this, though:

> fields is an array of objects.

I think that these are equivalent?

```yaml
array:
  - one
  - two
  - three
```

```yaml
array:
- one
- two
- three
```

Reply:
Wow, TIL. [Yaml Spec V1.2 §10.1.1.2.](https://yaml.org/spec/1.2.2/#10112-generic-sequence)